Mathlib/GroupTheory/Coset.lean

```diff
@@ -17,10 +17,6 @@ This file develops the basic theory of left and right cosets.
 
 ## Main definitions
 
-* `leftCoset a s`: the left coset `a * s` for an element `a : α` and a subset `s ⊆ α`, for an
-  `AddGroup` this is `leftAddCoset a s`.
-* `rightCoset s a`: the right coset `s * a` for an element `a : α` and a subset `s ⊆ α`, for an
-  `AddGroup` this is `rightAddCoset s a`.
```

Comment:
I think the module docstring should explain that `a • s` is a left coset and `op a • s` a right coset, using similar terminology to before; it just no longer belongs in a "definitions" section